trieste/acquisition/optimizer.py

```diff
@@ -82,8 +89,12 @@ def automatic_optimizer_selector(
         return optimize_discrete(space, target_func)
 
     elif isinstance(space, (Box, TaggedProductSearchSpace)):
-        num_samples = tf.minimum(NUM_SAMPLES_MIN, NUM_SAMPLES_DIM * tf.shape(space.lower)[-1])
-        return generate_continuous_optimizer(num_samples)(space, target_func)
+        num_samples = tf.maximum(NUM_SAMPLES_MIN, NUM_SAMPLES_DIM * tf.shape(space.lower)[-1])
+        num_runs = NUM_RUNS_DIM * tf.shape(space.lower)[-1]
```

Comment:
do we want to do something similar here `max(NUM_RUNS_MIN, NUM_RUNS_DIM * tf.shape(space.lower)[-1])`

then below in `generate_continuous_optimizer` we can also use `NUM_RUNS_MIN` as a default

Reply:
I think a min of 10 (i.e. 1 * 10) is fine for number of runs. Its just for the initial design where 1* 1000 wasn't enough and we needed to ensure > 5000.